test/e2e/lua/dynamic_configuration.go

```diff
@@ -52,8 +52,6 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(ing).NotTo(BeNil())
 
-		time.Sleep(10 * time.Second)
-
```

Comment:
This is not necessary because `WaitForNginxServer` below will make sure we wait enough after `ensureIngress` for the logs to appear.